system/services/src/com/percussion/services/content/impl/PSContentService.java

```diff
@@ -111,14 +112,18 @@ public List<PSKeyword> findKeywordsByLabel(String label, String sortProperty)
          if (StringUtils.isBlank(label))
             label = "%";
          
-         // get all requested keywords
-         Criteria criteria = session.createCriteria(PSKeyword.class);
-         criteria.add(Restrictions.like("label", label));
-         criteria.add(Restrictions.eq("keywordType", String.valueOf(1)));
-         if (!StringUtils.isBlank(sortProperty))
-            criteria.addOrder(Order.asc(sortProperty));
+
+      CriteriaBuilder builder = getSession().getCriteriaBuilder();
+      CriteriaQuery<PSKeyword> criteria = builder.createQuery(PSKeyword.class);
+      Root<PSKeyword> critRoot = criteria.from(PSKeyword.class);
```

Comment:
I don't think this will work.  You need to build a predicate list like in PSRecentDAO when you have multiple criteria